drs-auth_client/drs-auth_client.gemspec

```diff
@@ -0,0 +1,26 @@
+# coding: utf-8
+lib = File.expand_path("../lib", __FILE__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+require "drs/auth_client/version"
+
+Gem::Specification.new do |spec|
+  spec.name          = "drs-auth_client"
+  spec.version       = Drs::AuthClient::VERSION
+  spec.authors       = ["Jakub Novotny"]
+  spec.email         = ["novotny.jakub@gmail.com"]
```

Comment:
What's the guidance on gem author email addresses? shouldn't this be a MoJ email address?